evaluation/evaluate.py

```
@@ -565,12 +702,17 @@ def run_evaluation():
-    _save_clean_xlsx(xlsx_path, rows, collection_name, experiment_id, elapsed, timestamp)
-
-    # 원본 json 저장
+    # 원본 json 먼저 저장 (xlsx 가 실패해도 raw 결과는 보존되도록 순서 변경)
     with open(json_path, "w", encoding="utf-8") as f:
         json.dump(rows, f, ensure_ascii=False, indent=2)
+    print(f" - JSON 저장 완료: {json_path}")
+
+    # xlsx 저장 (실패해도 JSON/summary 는 영향 없음)
+    try:
+        _save_clean_xlsx(xlsx_path, rows, collection_name, experiment_id, elapsed, timestamp)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)